make/source.mk

```diff
@@ -263,67 +263,6 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             $(CMSIS_SRC) \
             $(DEVICE_STDPERIPH_SRC) \
 
-SIZE_OPTIMISED_SRC := $(SIZE_OPTIMISED_SRC) \
```

Comment:
Not sure if this should be done in this pull request.

Also, n.b. that we currently have 3 levels of optimisation: 'normal', 'size optimised', and 'speed optimised'. What is the reason to cut this down to 2?

Reply:
It is only for F1 and F3. It was to get it to build - as FLASH is overflowing. There is still 3 levels.

Reply:
In this case we should not remove this list, and the size optimisation on these files for >= F4.

Reply:
Size optimisation is not yet required for F4 or F7 (given the abundance of flash).

It is still available though, simply by adding an else here: https://github.com/betaflight/betaflight/pull/5045/files/dd9ed065358980bd76cd3a6f540e8e237f0be7b3#diff-55a0601892ff34962cfd5ee90c0eb0ccR321

and then filling in the files to be optimised.

Reply:
If we decide to stop size optimisation for >= F4, then I think this should be done in a separate pull request.